src/vm/proftoeeinterfaceimpl.cpp

```diff
@@ -4110,6 +4110,14 @@ DWORD ProfToEEInterfaceImpl::GetModuleFlags(Module * pModule)
 
         // Intentionally not checking for flat, since NGEN PEImages never have flat
         // layouts.
+
+#ifdef FEATURE_READYTORUN
```

Comment:
This is Kshama and I have an account @kspawa. Will look at this at this pr tomorrow. Thanks!